.gitignore

```diff
@@ -133,3 +133,7 @@ dmypy.json
 
 #Mac
 .DS_Store
+
+
+# NWB files
+**.nwb
```

Comment:
Can you propagate this up to the cookiecutter?

Reply:
done